frame/elections-phragmen/src/benchmarking.rs

```diff
@@ -228,7 +234,7 @@ benchmarks! {
 
 	submit_candidacy {
 		// number of already existing candidates.
-		let c in 1 .. T::MaxCandidates::get();
+		let c in 1 .. T::MaxCandidates::get() - 1;
```

Comment:
The reason why we subtract one here is very simple actually. It is because we submit `1..T::MaxCandidates::get()` before executing `_(RawOrigin::Signed(candidate_account.clone()), candidate_count::<T>())` where we also submit a candidacy. So we cannot have more than `T::MaxCandidates::get()` candidates so that is why we subtract one. @gpestana